src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -37,9 +37,9 @@
 
 import fault_config_utils
 import agent_utils
-import server_utils
 import write_host_file
 from daos_api import DaosContext, DaosLog, DaosApiError
+from server_utils import ServerManager, ServerFailed
```

Comment:
(pylint-import-error) Unable to import 'server_utils' (invalid syntax (<string>, line 397))